### ISSUE TITLE
Wikipages titles not searchable

### ISSUE THREAD

**Issue by brettp** [taigaio/taiga-back#440]
The WikiPage model doesn't include the title and there is no defined relationship between WikiPage and WikiLink, so titles are not included in searches.

**Comment by superalex** [comment]
You are right, we have a pending refactor related to wiki/titles and rich content.

Regards!,
